pkg/scheduler/apis/config/validation/validation_pluginargs_test.go

```diff
@@ -483,7 +484,7 @@ func assertErr(t *testing.T, wantErr string, gotErr error) {
 		if gotErr == nil {
 			t.Fatalf("\nwant err to be:\n\t%s\ngot:\n\tnil", wantErr)
 		}
-		if gotErr.Error() != wantErr {
+		if !errors.Is(gotErr, errors.New(wantErr)) {
```

Comment:
@krol3 the tests are failing, because the errors returned by the validation functions (`k8s.io/apimachinery/pkg/util/validation/field`) do not match (according to `errors.Is` spec) ones created here with `errors.New`.

@alculquicondor I'm not sure if we can leverage `errors.Is` here to check the type of the error, because `field.Error` doesn't implement  `interface{ Is(error) bool }`.  In this scenario `errors.Is` uses equality operator to compare pointers to `field.Error` structs. They are different as they are created dynamically and ultimately point to different variables.

Reply:
How about this : https://github.com/kubernetes/kubernetes/pull/95092

Reply:
> I'm not sure if we can leverage errors.Is here to check the type of the error, because field.Error doesn't implement interface{ Is(error) bool }

Then we can implement it :) I'm sure api-machinery folks will appreciate.

> How about this : #95092

I don't think so